### ISSUE TITLE
Colored filter on images with bluetooth gamedpad (UDLR)

### ISSUE THREAD

**Issue by freulaff** [tachiyomiorg/tachiyomi#5986]
### Steps to reproduce

When I use a bluetooth gamepad to go up or down when i'm reading a manga, the image appears sometimes with a colored tint/filter.

The color is related to the theme I have actually selected.
Blue theme gets a blue tint, tako theme gets a yellow tint,...

The issue is only appearing in the images, not the menus and only after I press on up/down/left/right on my gamepad. If I have the gamedpad conected but use the touchscreen for every move, the issue does not appear.

I use Tachiyomi with dark mode

### Expected behavior

That's the normal page
![Normal Screenshot_20210929-115216_Tachiyomi](https://user-images.githubusercontent.com/91600479/135247100-f324ebc7-21ff-433f-9c4c-a17155daf5c6.jpg)

### Actual behavior

Thats the page if I have the Apple Green theme:
![Green tint Screenshot_20210929-115419_Tachiyomi](https://user-images.githubusercontent.com/91600479/135247121-52648ab2-a7c0-4b6b-b8a2-0175416931e0.jpg)

That's the page if I have the Tako theme:
![Yellow tint Screenshot_20210929-115232_Tachiyomi](https://user-images.githubusercontent.com/91600479/135247369-e6a7282d-7f88-457c-8a33-03a10edb259f.jpg)

### Tachiyomi version

0.12.2

### Android version

Android 11

### Device

Samsung Galaxy Tab S6 Lite

### Other details

The problem doesn't appear when I use a bluetooth remote that emulate the volume up/down to change the pages.

The problem was with all the pages with version 0.12.1. Now, with the last version, it seems to appear randomly but at least 1-2x for every 30 pages

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by freulaff** [comment]
Apparently, I skimmed to quickly to see if the issue was already reported. This and #5973 are linked.
